### ISSUE TITLE
[1.12.2] Nutrients for potions and NBT

### ISSUE THREAD

**Issue by aks746** [WesCook/Nutrition#225]
Hello!

First of all thanks for the great mod! I use this all the time when playing Minecraft! 
Secondly, sorry if my english is bad.

Currently I am trying to add new types of nutrients to my modpack and I am having some problems. For example
I have three mods that add alcohol (Soot, Rustic, Growthcraft Community Edition) so it would be logical to add a separate nutrient "Alcohol" to add some negative effects for high values.

Growthcraft works well because the alcohol in this mod is food. But alcohol from the Rustic and Soot is a potions with NBT tags. 

Maybe you could add support for NBT-tagged potions and add the ability to add nutrients without food value? Or is it too complicated to implement?

**Comment by WesCook** [comment]
Hi aks,

You're right that Nutrition doesn't support NBT data right now.  Unfortunately I'm not currently working on the mod, so I'm not developing features like that.  I'd like to have NBT support, though it would be complex with the current JSON files.

Realistically, a feature like that would only emerge in a 1.16 port.  It'd be more essential in the new versions because metadata has been removed.  Theoretically at that point it could be backported, but that's a lot of big requirements that aren't likely to be worked on soon.

So unfortunately all I can say is that NBT foods aren't currently supported.  I apologize for the frustration, as I've had to deal with that problem a few times myself.
